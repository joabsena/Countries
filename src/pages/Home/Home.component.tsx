import * as Styles from './Home.styles'

export function Home() {
  return <Styles.Wrapper>hello world</Styles.Wrapper>
}
