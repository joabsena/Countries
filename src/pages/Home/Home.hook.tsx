import { useState } from 'react'

export const useHome = () => {
  const [state] = useState()

  return {
    state,
  }
}
