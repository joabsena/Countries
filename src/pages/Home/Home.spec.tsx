import { describe, expect, it } from 'vitest'
import { render } from '@testing-library/react'
import { Home } from './Home.component'

describe('Page: Home', () => {
  it('should render snapshot', () => {
    const { container } = render(<Home />)

    expect(container).toMatchSnapshot()
  })
})
