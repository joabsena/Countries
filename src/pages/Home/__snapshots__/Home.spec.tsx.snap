// Vitest Snapshot v1

exports[`Page: Home > should render snapshot 1`] = `
<div>
  <div>
    hello world
  </div>
</div>
`;
